index.html

```diff
@@ -246,11 +246,24 @@ <h3 class="sub-social-media ">LET’S CONNECT</h3>
                         <hr>
                     </li>
                 </ul>
-
             </div>
-
-
         </section>
+        <!-- contact form -->
+        <article class="contact-me">
+            <h3 class="contact-title">Contact me</h3>
+            <p class="contact-text">If you have an application you are interested in developing, a feature that you need built or a project
+                that needs coding.
+                I’d love to help with it!</p>
+          <div class="form-container">
+                <form action="https://formspree.io/f/moqzkvjg" method="POST">
+                <input class="first-name" type="text" name="fullname" maxlength="30" minlength="2" placeholder="Enter your name" required>
+                <input class="first-name" type="email" name="email" placeholder="Enter your email" required>
+                <textarea  class="contact-textarea" name="message" id="" cols="33" rows="10" placeholder="Write your message here..." maxlength="500"></textarea>
```

Comment:
- [ ] As mentioned in the requirements all fields should be required, currently, there is no required attribute present in the textarea element, this makes it possible to submit empty forms without any message, please add the `required` attribute here